### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: screens/Dashboard.tsx [hunk 1]
```diff
@@ -63,171 +66,204 @@ export default function Dashboard({ navigation, route }: Props) {
...
 	
 
-	useEffect(() => {
-		const backHandler = BackHandler.addEventListener(
-			'hardwareBackPress',
-			backAction
-		);
-		return () => backHandler.remove();
-	}, []);
+	// useEffect(() => {
```
Comment (teebarg) [inline screens/Dashboard.tsx]: can we remove if not in use?

File: screens/EditProfile.tsx [hunk 2]
```diff
@@ -64,10 +68,27 @@ export default function Dashboard({ navigation, route }: Props) {
...
+  async function handleUpdate (){
+    try {
+      let result = await axios({
+        method: 'PATCH',
+        url: '/customers',
+        headers: { 'Authorization': `Bearer ${state.token}` },
+        data: user
+      });
+    } catch (error) {
+      console.log(error.response.data)
```
Comment (teebarg) [inline screens/EditProfile.tsx]: remove console

File: screens/EditProfile.tsx [hunk 3]
```diff
@@ -64,10 +68,27 @@ export default function Dashboard({ navigation, route }: Props) {
...
+    }
+    let result = await axios({
+      method: 'PATCH',
+      url: '/customers',
+      headers: { 'Authorization': `Bearer ${state.token}` },
+      data: user
+    });
+    if(result){
+      console.log(result.data)
```
Comment (teebarg) [inline screens/EditProfile.tsx]: remove log

File: screens/Dashboard.tsx (hunk 1 above)
Comment (darmhoo) [reply screens/Dashboard.tsx]: doone

**Review by teebarg — APPROVED** [review]

**Pull request merged**